DatabaseAdapters/Repositories/MeasurementRepository.cs

```diff
@@ -33,10 +33,13 @@ public async Task<List<AggregatedMeasurement>> GetAggregatedHistoryForSensorAsyn
     DateTime endDate,
     MeasurementHistoryGranularity granularity, long sensorId) {
 
-    return await GroupAndAggregate(databaseContext.Measurements.Where(m =>
-                   m.Timestamp >= startDate && m.Timestamp < endDate && m.SensorId == sensorId), granularity)
+    var measurements = await databaseContext.Measurements
```

Comment:
Do not use var, dynamic or object unless required. Refactor all parts of code to use either explicit typing or Type variable = new() instead of var variable = new Type()